.github/workflows/ci.yaml

```diff
@@ -213,6 +213,69 @@ jobs:
         #  testlib modules of gax
         run: mvn package clirr:check -DskipTests -Dfmt.skip
 
+  build-java8-showcase:
+    name: "build(8) for showcase"
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: maven
+      - name: Install all modules using Java 11
+        shell: bash
+        run: |
+          mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip -Dfmt.skip
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: temurin
+      - run: java -version
+      - name: Showcase golden tests
```

Comment:
The golden tests verify the static generation results, which would not change with Java 8.

Reply:
Ah that's a good point, thank you